src/pip/_internal/commands/download.py

```diff
@@ -115,6 +116,15 @@ def __init__(self, *args, **kw):
                   "this option."),
         )
 
+        cmd_opts.add_option(
+            '--json',
+            dest='json',
+            action='store_true',
+            default=False,
+            help=("Output information about affected packages "
+                  "in an easily parseable json format"),
```

Comment:
"affected"? Surely "downloaded", as this is only for `pip download`? Also, describing JSON as "easily parseable" is unnecessary...